src/go/guestagent/pkg/types/portmapping.go

```diff
@@ -23,9 +23,12 @@ import "github.com/docker/go-connections/nat"
 type PortMapping struct {
 	// Remove indicates whether the port mappings should be removed (true) or added (false)
 	Remove bool `json:"remove"`
-	// Ports contains the port mappings for both IPv4 and IPv6 addresses
+	// Ports contains the port mappings for both IPv4 and IPv6 addresses.  The host address
```

Comment:
Just to clarify, the term "host" always refers to the actual host machine, such as a Windows machine, and cannot mean anything else. If there's any confusion regarding the host being interpreted as a previous network hop—like a namespace network, that would be the upstream/downstream address.